packages/components/src/atoms/PaneCloseIcon/PaneCloseIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import * as S from "./PaneCloseIcon.styled";
+
+export type PaneCloseIconType = {
+  onClick: () => void;
+  containerId?: string;
+  containerStyle?: React.CSSProperties;
+  iconStyle?: React.CSSProperties;
+};
+
+export const PaneCloseIcon: React.FC<PaneCloseIconType> = (props) => {
+  const { containerId = "pane-close", containerStyle = {}, iconStyle = {}, onClick } = props;
+
+  return (
+    <S.PaneCloseIconContainer id={containerId} style={{ ...containerStyle }} onClick={onClick}>
+      <S.DoubleLeftOutlined style={{ ...iconStyle }} />
+    </S.PaneCloseIconContainer>
+  );
+};
```

Comment:
Third discussion I'd like to have is colocating style instead of having a `.styled.tsx`. The downside is that your file might become bigger but my preferred solution here is to make smaller React components instead of extracting the style on the side.

Biggest benefit is that if a style is unused your IDE will tell it to you, in .styled.tsx we export everything and it will never give you that intel as a result.

Reply:
The main reason I went for a separate file is just for structuring purposes. I like having them in separate files as it's easier to follow when refactoring/debugging things ( instead of navigating through a really big file ).

But fair point about the unused styles that the IDE won't notice when they are in a separate file. @devcatalin @monojack wdyt?